specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3258,6 +3258,76 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/blobUris": {
+      "get": {
+        "tags": [ "Reports" ],
+        "description": "Gets all Risky IP report URIs for the last 7 days.",
+        "x-ms-examples": {
+          "services_listAllRiskyIpDownloadReport": {
+            "$ref": "./examples/RiskyIpDownload.json"
+          }
+        },
+        "operationId": "services_listAllRiskyIpDownloadReport",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },          
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Risky IP report URIs for the last 7 days.",
+            "schema": {
+              "$ref": "#/definitions/RiskyIPBlobUris"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/blobUri": {
+      "post": {
+        "tags": [ "Reports" ],
+        "description": "Initiate the generation of a new Risky IP report. Returns the URI for the new one.",
+        "x-ms-examples": {
+          "services_listCurrentRiskyIpDownloadReport": {
+            "$ref": "./examples/RiskyIpDownload.json"
+          }
+        },
+        "operationId": "services_listCurrentRiskyIpDownloadReport",

```

Comment:
operationId should not be camel-cased.

Reply:
I'll make the change. It seems I'll have to fix it for the whole document though. Is there any implication to the spec for changing it everywhere?

Reply:
I'd leave it as-is then for consistency with the rest of the docs.  It's highly likely that there's zero impact to codegen as most generators probably fix the casing.

Reply:
Great. So are we good to go with this PR?

Reply:
Once @KrisBash signs off from ARM side I will complete my review.

Reply:
Thank you, Joel.

Reply:
@KrisBash, could you please take a look?